### PR TITLE
Update main-procedure.md

### PR DESCRIPTION
**Pull request by Youssef1313**
- Added vb language to code snippets.
- Removed extra spaces.

### CODE REVIEW

**Comment by Youssef1313** [comment]
I think the article is missing information about asynchronous main methods ?
I don't have enough information about starting from which version it is supported to add some note in the article.

File: docs/visual-basic/programming-guide/program-structure/main-procedure.md [hunk 1]
```diff
@@ -10,97 +10,97 @@ helpviewer_keywords:
...
-                    ' Insert code to examine cmdArgs(argNum) and take  
-                    ' appropriate action based on its value.  
-                Next argNum  
-            End If  
-            ' Insert call to appropriate starting place in your code.  
-            ' On return, assign appropriate value to returnValue.  
-            ' 0 usually means successful completion.  
+- `Main` can also take a `String` array as an argument. Each string in the array contains one of the command-line arguments used to invoke your program. You can take different actions depending on their values.
+
+    ```vb
```
Comment (Thraka) [inline docs/visual-basic/programming-guide/program-structure/main-procedure.md]: There are a few more double spaces at the end of the lines of this block, can you fix those too?

File: docs/visual-basic/programming-guide/program-structure/main-procedure.md [hunk 2]
```diff
@@ -10,97 +10,97 @@ helpviewer_keywords:
...
+    Module mainModule
+        Function Main() As Integer
+            MsgBox("The Main procedure is starting the application.")
+            Dim returnValue As Integer = 0
+            ' Insert call to appropriate starting place in your code.
+            ' On return, assign appropriate value to returnValue.
+            ' 0 usually means successful completion.
             MsgBox("The application is terminating with error level " &
```
Comment (Thraka) [inline docs/visual-basic/programming-guide/program-structure/main-procedure.md]: There are a few more double spaces at the end of the lines of this block, can you fix those too?

File: docs/visual-basic/programming-guide/program-structure/main-procedure.md [hunk 3]
```diff
@@ -10,97 +10,97 @@ helpviewer_keywords:
...
-  
-- `Main` can also take a `String` array as an argument. Each string in the array contains one of the command-line arguments used to invoke your program. You can take different actions depending on their values.  
+                 CStr(returnValue) & ".")
+            Return returnValue
+        End Function
+    End Module
+    ```
   
-    ```  
-    Module mainModule
```
Comment (Thraka) [inline docs/visual-basic/programming-guide/program-structure/main-procedure.md]: There are a few more double spaces at the end of the lines of this block, can you fix those too?

**Review by Thraka — CHANGES_REQUESTED** [review]
Thanks a bunch for cleaning this topic. Those double spaces were automatically created when we migrated this content from one system to another. So thanks!

Can you do a few more? I pointed them out. Thanks!

**Review by Thraka — APPROVED** [review]

**Pull request merged**